python/python/bystro/utils/config.py

```diff
@@ -41,3 +42,21 @@ def get_opensearch_config() -> ConfigDict:
     with opensearch_config_filepath.open() as search_config_file:
         config_dict: ConfigDict = YAML(typ="safe").load(search_config_file)
     return config_dict
+
+
+# define the set of permissible reference genomes so that they're
+# available both statically and at runtime
+ReferenceGenome = Literal["hg19", "hg38"]
```

Comment:
This is fine as a starting point. We do support non-human genomes, and have mapping configs for those, but I am ok with focusing on humans for the initial re-release of bystro, so non-blocking, and I appreciate the mapping config checks